docs/local_development.md

```diff
@@ -209,6 +217,17 @@ The `docker-compose.yml` file describes the 3 services that the project needs to
 - [Intro to Docker](https://www.revsys.com/tidbits/brief-intro-docker-djangonauts/): Lacey wrote a good intro tutorial to Docker and Django, without Harry Potter metaphors this time :),
 - [Jérôme Petazzoni's training slides and talks](https://container.training/): presentations and slides if you want to dive into Docker.
 
+#### Useful commands using Django with Docker
+
+To open a terminal session inside the docker container (`docker container ls` to see active docker container ids):
+`docker exec it {docker-container-id} bash`
+Activate the python environment:
+`source dockerpythonvenv/bin/activate`
+
+Then you can do Django stuff like:
+`cd network-api/ && python manage.py makemigrations --merge`
```

Comment:
@cadecairos I see! I couldn't get invoke to work with those flags, but I see that I was trying `inv "makemigrations --merge"`

In which case it might be better to update the docs with your example command. I think @KalobTaulien and I have both missed this, which seems like enough of a reason to include it. :-D

Reply:
Would you mind adding that to this PR or will you file it as a follow-up?

Reply:
doing it right now!